dns-dodo.service

```diff
@@ -0,0 +1,10 @@
+[Unit]
+Description=The dns-dodo DNS record updater
+Requires=network-online.target
+After=network-online.target
+
+[Service]
+ExecStart=/usr/bin/dns-dodo update-dns -p
```

Comment:
Ahh damn, I thought I'd gotten away with it! 😉

The reason I put `/usr/bin` is that once the unit file is in I'd like to provide a `PKGBUILD` file for Arch Linux AUR which would install to `/usr/bin`. However, that would mean having to package for each distro individually. I will investigate using systemd's user units in combination with `$GOPATH` for a cross-distro solution.